src/component/Home.js

```diff
@@ -1,9 +1,14 @@
-import React from 'react';
+import React from "react";
+import GetToken from './GetToken';
 
 function Home() {
-	return (
-		<h1>congrats this is the home page</h1>
-	)
+  return (
+    <div>
+      <h1>New Shopping List</h1>
+      <GetToken />
+      <p><a href="/join">Join Existing List</a></p>
+    </div>
+  );
 }
 
-export default Home
+export default Home;
```

Comment:
Ethan and I ended up deleting the Home component, since it's not needed. When we merge, we should make sure this gets deleted.

Reply:
So I disagree with deleting the `Home` component actually - and here's why. 

The initial landing page has to have two links/buttons for the two options - either start a new list, or join an existing list. Currently, you two were able to delete Home, because your branch routes directly to `GetToken`, but that component doesn't have a link to the `JoinList` component. 

I suppose we could add the link to the `JoinList` component in the `GetToken` component, but I feel like it's a little more clear to bring those ideas together into the `Home` component. That's just my opinion though - other advantages/disadvantages 🤷‍♀I'd love to hear some more thoughts on this!

Reply:
i think we should leave in home as well and agree with what Mindy was saying. the home component is the landing page where you get to chose between joining an existing list and creating a new list (by creating a token). the add item component should not come into play until join or create has happened.

Reply:
@gunderodd